app/components/results-list/component.js

```diff
@@ -50,11 +50,11 @@ export default Ember.Component.extend({
             let queryParams = {};
             let page = Number(this.parameters["page"]);
             if (!page) {
-                page = 0;
+                page = 1;
             }
-            page = page-10;
-            if (page < 0) {
-                page = 0;
+            page = page-1;
```

Comment:
I think it makes more sense to first check if page > 1 before subtracting 1 here